[2023-03-29 13:42:51,034] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.transform_data_task manual__2023-03-29T13:42:24.222819+00:00 [queued]>
[2023-03-29 13:42:51,152] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.transform_data_task manual__2023-03-29T13:42:24.222819+00:00 [queued]>
[2023-03-29 13:42:51,154] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 13:42:51,156] {taskinstance.py:1239} INFO - Starting attempt 1 of 3
[2023-03-29 13:42:51,157] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 13:42:51,232] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): transform_data_task> on 2023-03-29 13:42:24.222819+00:00
[2023-03-29 13:42:51,256] {standard_task_runner.py:52} INFO - Started process 14830 to run task
[2023-03-29 13:42:51,286] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'SP_500_DATA_PIPELINE_v2', 'transform_data_task', 'manual__2023-03-29T13:42:24.222819+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpojfx7_4_', '--error-file', '/tmp/tmp7mky2gj2']
[2023-03-29 13:42:51,300] {standard_task_runner.py:77} INFO - Job 119: Subtask transform_data_task
[2023-03-29 13:42:51,619] {logging_mixin.py:109} INFO - Running <TaskInstance: SP_500_DATA_PIPELINE_v2.transform_data_task manual__2023-03-29T13:42:24.222819+00:00 [running]> on host 5ebc180c7cc0
[2023-03-29 13:42:51,943] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=SP_500_DATA_PIPELINE_v2
AIRFLOW_CTX_TASK_ID=transform_data_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T13:42:24.222819+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T13:42:24.222819+00:00
[2023-03-29 13:44:30,806] {stock_data_transform.py:135} ERROR - An error occurred during transformation: Recursive view `stock_data` detected (cycle: `stock_data` -> `stock_data`)
[2023-03-29 13:45:33,011] {python.py:175} INFO - Done. Returned value was: None
[2023-03-29 13:45:33,072] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=SP_500_DATA_PIPELINE_v2, task_id=transform_data_task, execution_date=20230329T134224, start_date=20230329T134251, end_date=20230329T134533
[2023-03-29 13:45:33,195] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-29 13:45:33,348] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
